test/e2e_flags.go

```diff
@@ -40,23 +38,15 @@ type EventingEnvironmentFlags struct {
 func initializeEventingFlags() *EventingEnvironmentFlags {
 	var f EventingEnvironmentFlags
 
-	defaultRepo := os.Getenv("KO_DOCKER_REPO")
-
-	if defaultRepo == "" {
-		defaultRepo = os.Getenv("KO_DOCKER_REPO")
-	}
-
-	flag.StringVar(&f.DockerRepo, "dockerrepo", defaultRepo,
-		"Provide the uri of the docker repo you have uploaded the test image to using `uploadtestimage.sh`. Defaults to $KO_DOCKER_REPO")
-
-	flag.StringVar(&f.Tag, "tag", "e2e", "Provide the version tag for the test images.")
-
 	flag.StringVar(&f.Provisioner, "clusterChannelProvisioner", "in-memory-channel", "The name of the Channel's clusterChannelProvisioner. Only the in-memory-channel is installed by the tests, anything else must be installed before the tests are run.")
 
 	flag.Parse()
 
 	logging.InitializeLogger(pkgTest.Flags.LogVerbose)
 
+	f.DockerRepo = pkgTest.Flags.DockerRepo
+	f.Tag = pkgTest.Flags.Tag
```

Comment:
Did this change due to changes in knative/pkg?

Reply:
yes, the changes came along too